### PR TITLE
Add whether columns have a index

### PR DESCRIPTION
**Pull request by yuh-pen**
Add a item show whether columns have a index

### CODE REVIEW

File: lib/schemate.rb [hunk 1]
```diff
@@ -35,11 +35,12 @@ def export_csv
 
           next unless tables.include?(table_name)
           file << %W(#{table_name})
-          file << %w(FieldName Attitutde Size NULL Default Comment)
+          file << %w(FieldName Attitutde Size NULL Default Index Comment)
           model.columns.each do |columns|
+            index = '  ○' if model.connection.index_exists?(:"#{table_name}", :"#{columns.name}")
```
Comment (eishisaito) [inline lib/schemate.rb]: could you replace ◯ to true?

File: lib/schemate.rb [hunk 2]
```diff
@@ -56,10 +57,11 @@ def export_md
           next unless tables.include?(table_name)
           file.puts("# Table name: `#{table_name}` \n")
           file.puts('## Columns')
-          file.puts('|FieldName |Attitutde |Size |NULL |Default |Comment |')
-          file.puts('|---|---|---|---|---|---|')
+          file.puts('|FieldName |Attitutde |Size |NULL |Default |Index |Comment |')
+          file.puts('|---|---|---|---|---|---|---|')
           model.columns.each do |columns|
-            file.puts("|#{columns.name} |#{columns.type} |#{columns.limit} |#{columns.null} |#{columns.default} |#{columns.comment}|")
+            index = '○' if model.connection.index_exists?(:"#{table_name}", :"#{columns.name}")
```
Comment (eishisaito) [inline lib/schemate.rb]: same here!

**Comment by yuh-pen** [comment]
I replaced circles to 'true'.
Please review.

**Comment by eishisaito** [comment]
looks good! thank you!

**Pull request merged**